include/lsst/afw/geom/detail/frameSetUtils.h

```diff
@@ -0,0 +1,113 @@
+// -*- lsst-c++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2017 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_AFW_GEOM_FRAMESETUTILS_H
+#define LSST_AFW_GEOM_FRAMESETUTILS_H
+
+#include <memory>
+#include <vector>
+
+#include "astshim.h"
+#include "ndarray.h"
+
+#include "lsst/afw/geom/Endpoint.h"
+#include "lsst/afw/geom/Transform.h"
+#include "lsst/daf/base/PropertyList.h"
+
+namespace lsst {
+namespace afw {
+namespace geom {
+namespace detail {
```

Comment:
I hope these will not be widely used. I view these as I/O details users should never see.